### ISSUE TITLE
Web client: Text is cut off in numbered list with long string 

### ISSUE THREAD

**Issue by sCmEcOLJUcL** [mattermost/mattermost#27104]
#### Summary
Web client: Text is cut off in numbered list with long string 

#### Steps to reproduce
Example message 
```
1. aaaaaaaaaa - 11227209,11227234,11227210,11227211,11227213,11227231,11227229,11227230,11227232,11227235,11227212,11227233,11227209,11227234,11227210,11227211,11227213,11227231,11227229,11227230,11227232,11227235,11227212,11227233,11227209,11227234,11227210,11227211,11227213,11227231,11227229,11227230,11227232,11227235,11227212,11227233,11227209,11227234,11227210,11227211,11227213,11227231,11227229,11227230,11227232,11227235,11227212,11227233,11227209,11227234,11227210,11227211,11227213,11227231,11227229,11227230,11227232,11227235,11227212,11227233
2. bbbbbbbbbbbbbb - 11227209,11227234,11227210,11227211,11227213,11227231,11227229,11227230,11227232,11227235,11227212,11227233
3. cccccccccccc  - qweeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
```
#### Expected behavior
Mattermost ver. 9.7.4/9.31, see screenshot

![ksnip_20240522-162152](https://github.com/mattermost/mattermost/assets/29278617/3a2023f4-2de2-4e20-885b-9d2d011b424a)

**Comment by sCmEcOLJUcL** [comment]
Hi! please tell me what releases consiste the fix?
I checked 9.9.1

**Comment by amyblais** [comment]
> Hi! please tell me what releases consiste the fix? I checked 9.9.1

The fix is queued for v9.10 (July 16th) release.
